domains/abby.json

```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "MiloDev123",
+        "email": "milo@milosantos.com",
+        "discord": "699318362848886864"
+    },
+    "record": {
+        "CNAME": "mx.milosantos.com"
```

Comment:
This doesn't display anything, also appears to likely be a MX record, you might need to recheck these records.

Reply:
the ignore the "mx" name, its the only record that always has the dynamic ip updated, by thr way, what do you mean by "it doesnt display anything"?